### PR TITLE
Henri/issue 40/contact

### LINKED ISSUE #38: Contact List UI
A list of all of the contacts that are configured. This will be displayed on the 'Messages' page as the left 1/3 of the page.

Clicking on a contact will open that message history

### LINKED ISSUE #40: New Contact Input Box
This will be an input box component at the top of the contact list that will allow the user to create a new connection with a contact.

### PR DESCRIPTION
**Pull request by Vanderscycle**
The contact route has been created and can accurately query the backend

resolves #40 
resolves #38

### CODE REVIEW

**Comment by Vanderscycle** [comment]
issue #40 related

File: frontend/src/lib/NewContacts.svelte [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      </div>
+      <form class="mt-8 space-y-6"  on:submit|preventDefault={newContactValidator}>
+        <input type="hidden" name="remember" value="true">
+        <div class="rounded-md shadow-sm -space-y-px">
+          <div>
+            <select
+              bind:value={localVal[0]}
+              required class="appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-t-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm"
+              >
+              <!-- BUG: not assignign the proper value-->
```
Comment (brianorwhatever) [inline frontend/src/lib/NewContacts.svelte:83]: still a bug?

**Review by brianorwhatever — APPROVED** [review]
looks good to me. merge when ready

File: frontend/src/lib/NewContacts.svelte (hunk 1 above)
Comment (Vanderscycle) [reply frontend/src/lib/NewContacts.svelte:83]: yeah, not sure why Select isn't binding the value properly to the local array. The value it is supposed to replace is set to 'web as default.

**Comment by brianorwhatever** [comment]
so this is why i would like to keep issues and PRs separate. I would like to merge in the input box feature and allow you to continue to work on the other issue. In the future we should create a new branch for working on a related feature. If there is code from the other feature that is required (like in this example) we can create a branch off of the PR branch.

Let's not worry about it for this instance and just merge this in once the contact list functionality is ready to go.

**Comment by Vanderscycle** [comment]
Sounds good, I'll pay more attention to the branch management.

**Pull request merged**